tests/testsuite/manifest.go

```diff
@@ -114,6 +114,7 @@ func DeleteRawManifest(object unstructured.Unstructured) error {
 	policy := metav1.DeletePropagationBackground
 	options := &metav1.DeleteOptions{PropagationPolicy: &policy}
 
+	fmt.Printf(fmt.Sprintf("Calling DELETE on testing manifest: %s\n", uri))
```

Comment:
Debug print?

Reply:
I was hoping we can keep this, very useful information when combined with the timestamp

Reply:
You can use our default logger instead. It would then be associated with the test if it fails and we don't see it always printed, kepping the build log more crisp.

Reply:
+1 for Romans suggestion. I don't oppose this debug print otherwise.

Reply:
Done